server/channels/api4/system.go

```diff
@@ -1034,7 +1079,9 @@ func getAppliedSchemaMigrations(c *Context, w http.ResponseWriter, r *http.Reque
 		return
 	}
 
-	w.Write(js)
+	if _, err := w.Write(js); err != nil {
+		c.Logger.Warn("Error while writing response", mlog.Err(err))
+	}
 	auditRec.Success()
```

Comment:
You can move the `auditRec.Success()` before the write statement

Reply:
Done!